src/constants/index.ts

```diff
@@ -18,6 +18,8 @@ export const BACKDROP_OBJECT_OPACITY_MAX = 100;
 export const BACKDROP_OBJECT_OPACITY_DEFAULT = 50;
 export const COLOR_RAMP_RANGE_DEFAULT: [number, number] = [0, 0];
 
+export const CANVAS_ASPECT_RATIO = 14.6 / 10;
```

Comment:
does this need to be kept in sync with other magic numbers elsewhere?

Reply:
This is the only magic number now!